docs/source/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
  - Added internal_index_image_copy and internal_index_image_copy_resolved attributes support
+ - Added RecursiveRelatedBundlesModel 
```

Comment:
@xDaile This should be moved to the Unreleased section
v6.0.0 was already released on the date mentioned with the internal_index_copy change